taxi/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+    class Meta:
+        ordering = ('name', 'country',)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer,
+                                     on_delete=models.CASCADE,
+                                     related_name='cars')
+
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name='cars')
```

Comment:
According to the checklist, it's better to use settings.AUTH_USER_MODEL instead of AbstractUser. This will allow you to change the user model in one place in the future if needed.